### PR TITLE
python3Packages.markupsafe: update pname

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
- Update pname
- Update homepage
- Add `pythonImportsCheck`

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 129128` at c5a562e8 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>71 packages marked as broken and skipped:</summary>
  <ul>
    <li>adobe-reader</li>
    <li>adoptopenjdk-hotspot-bin-13</li>
    <li>adoptopenjdk-hotspot-bin-14</li>
    <li>adoptopenjdk-jre-hotspot-bin-13</li>
    <li>adoptopenjdk-jre-hotspot-bin-14</li>
    <li>adoptopenjdk-jre-openj9-bin-13</li>
    <li>adoptopenjdk-jre-openj9-bin-14</li>
    <li>adoptopenjdk-openj9-bin-13</li>
    <li>adoptopenjdk-openj9-bin-14</li>
    <li>airtame</li>
    <li>autotrace</li>
    <li>cassandra_2_1</li>
    <li>cassandra_2_2</li>
    <li>clickshare-csc1</li>
    <li>curaByDagoma</li>
    <li>cura_stable</li>
    <li>displaycal</li>
    <li>dvd-slideshow</li>
    <li>electron_3</li>
    <li>electron_4</li>
    <li>electron_5</li>
    <li>epsxe</li>
    <li>erlang_basho_R16B02</li>
    <li>evilvte</li>
    <li>gegl_0_4</li>
    <li>gnash</li>
    <li>gnome.gegl_0_4</li>
    <li>googleearth-pro</li>
    <li>grass</li>
    <li>green-pdfviewer</li>
    <li>hadoop</li>
    <li>hadoop_2_7</li>
    <li>hadoop_2_8</li>
    <li>hadoop_2_9</li>
    <li>hplipWithPlugin_3_16_11</li>
    <li>hplip_3_16_11</li>
    <li>impressive</li>
    <li>invoice2data</li>
    <li>jboss</li>
    <li>k2pdfopt</li>
    <li>keyfinder</li>
    <li>linuxPackages_hardkernel_4_14.akvcam</li>
    <li>linuxPackages_hardkernel_4_14.virtualbox</li>
    <li>linuxPackages_hardkernel_latest.akvcam</li>
    <li>linuxPackages_hardkernel_latest.virtualbox</li>
    <li>loxodo</li>
    <li>metamorphose2</li>
    <li>obinskit</li>
    <li>odpdown</li>
    <li>opentsdb</li>
    <li>pcsxr</li>
    <li>pdfdiff</li>
    <li>pdfread</li>
    <li>pgadmin</li>
    <li>pig</li>
    <li>plover.stable</li>
    <li>riak</li>
    <li>scribus</li>
    <li>scyther</li>
    <li>spark</li>
    <li>staruml</li>
    <li>stepmania</li>
    <li>termplay</li>
    <li>torchat</li>
    <li>tvheadend</li>
    <li>twinkle</li>
    <li>unicap</li>
    <li>vdrPlugins.vaapidevice</li>
    <li>vdrPlugins.xineliboutput</li>
    <li>winpdb</li>
    <li>xpdf</li>
  </ul>
</details>
<details>
  <summary>14428 packages skipped due to time constraints:</summary>
  <ul>
    <li>AusweisApp2</li>
    <li>DisnixWebService</li>
    <li>EBTKS</li>
    <li>MIDIVisualizer</li>
    <li>OSCAR</li>
    <li>R</li>
    <li>SDL_Pango</li>
    <li>SDL_mixer</li>
    <li>Sylk</li>
    <li>_1password-gui</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>14 packages built successfully:</summary>
  <ul>
    <li>djvulibre</li>
    <li>gdk-pixbuf</li>
    <li>gi-docgen</li>
    <li>librsvg (gnome.librsvg)</li>
    <li>pango (gnome2.pango)</li>
    <li>imagemagick</li>
    <li>python38Packages.Mako</li>
    <li>python38Packages.jinja2</li>
    <li>python38Packages.markupsafe</li>
    <li>sphinx (python38Packages.sphinx)</li>
    <li>python38Packages.typogrify</li>
    <li>python39Packages.jinja2</li>
    <li>python39Packages.markupsafe</li>
    <li>python39Packages.sphinx</li>
  </ul>
</details>

**Comment by dotlambda** [comment]
Can you rebase on staging?

File: pkgs/development/python-modules/markupsafe/default.nix [hunk 1]
```diff
@@ -6,24 +6,26 @@
 }:
 
 buildPythonPackage rec {
-  pname = "MarkupSafe";
+  pname = "markupsafe";
```
Comment (mweinelt) [inline pkgs/development/python-modules/markupsafe/default.nix:9]: https://github.com/pallets/markupsafe/blob/main/setup.cfg#L2

Comment (dotlambda) [reply pkgs/development/python-modules/markupsafe/default.nix:9]: What do you mean?

Comment (mweinelt) [reply pkgs/development/python-modules/markupsafe/default.nix:9]: For the capitalization of the pname we use the name attribute from the setup.py/cfg metadata AFAIK.

Comment (dotlambda) [reply pkgs/development/python-modules/markupsafe/default.nix:9]: No, we normalize it according to PEP 0503: https://nixos.org/manual/nixpkgs/unstable/#contributing

Comment (mweinelt) [reply pkgs/development/python-modules/markupsafe/default.nix:9]: Ah okay, I thought that was only for attributes. Thanks!

**Pull request merged**